os/x11/window.h

```diff
@@ -108,6 +108,8 @@ class WindowX11 : public Window {
   bool setX11Cursor(::Cursor xcursor);
   bool requestX11FrameExtents();
   void getX11FrameExtents();
+  XEvent checkPrevKeyEvent() { return m_prev_key_event; }
+  void setPrevKeyEvent(XEvent curr_key_event) { m_prev_key_event = curr_key_event; }
```

Comment:
We can remove these private methods and access the member directly.

Reply:
Done as suggested.